hub/management/commands/import_christian_aid_group_locations.py

```diff
@@ -47,6 +47,8 @@ class Command(BaseAreaImportCommand):
         "default_value": 0,
         "is_filterable": True,
         "comparators": DataSet.numerical_comparators(),
+        "unit_type": "raw",
+        "unit_distribution": "people_in_area",
```

Comment:
I don't think "people_in_area" is correct here. I think "point" is possibly most accurate, but I suspect we possibly need some more categories. There's more of these further down.